test/index.js

```diff
@@ -764,6 +777,11 @@ test('traverse', function() {
   eq(Z.traverse(Identity, identInc, [1, 2, 3]), Identity([2, 3, 4]));
   eq(Z.traverse(Identity, identInc, Nil), Identity(Nil));
   eq(Z.traverse(Identity, identInc, Cons(1, Cons(2, Cons(3, Nil)))), Identity(Cons(2, Cons(3, Cons(4, Nil)))));
+
+  assert.doesNotThrow(function() {
+    // traverse and run the accumulator. this should stack safe.
+    Z.traverse(StateL[FL.of], subAccum, range(70000))._f(100);
+  }, RangeError);
```

Comment:
The documentation is somewhat confusing, but it seems we needn't provide this argument.